test/BUILD.gn

```diff
@@ -68,6 +68,12 @@ test("brave_browser_tests") {
     "//brave/components/brave_shields/browser/ad_block_service_browsertest.cc",
     "//brave/components/brave_shields/browser/https_everywhere_service_browsertest.cc",
     "//brave/renderer/brave_content_settings_observer_browsertest.cc",
+    "//chrome/browser/extensions/browsertest_util.cc",
+    "//chrome/browser/extensions/browsertest_util.h",
+    "//chrome/browser/extensions/extension_browsertest.cc",
+    "//chrome/browser/extensions/extension_browsertest.h",
+    "//chrome/browser/extensions/updater/extension_cache_fake.cc",
+    "//chrome/browser/extensions/updater/extension_cache_fake.h",
   ]
   defines = [ "HAS_OUT_OF_PROC_TEST_RUNNER" ]
```

Comment:
I couldn't add a comment below, but did you remove `test/data/adblock-data/3/ABPFilterParserData.dat` ?
I think we can remove that and use `test/data/adblock-data/ABPFilterParserData.dat` or you can just use the original file, it would probably be best for download size of the repo to just move it or just reference the original path.